packages/manager/src/features/CloudPulse/Widget/CloudPulseWidget.tsx

```diff
@@ -299,6 +299,9 @@ export const CloudPulseWidget = (props: CloudPulseWidgetProperties) => {
             padding={1}
           >
             <Typography
+              data-qa-widget_header={convertStringToCamelCasesWithSpaces(
```

Comment:
```suggestion
              data-qa-widget-header={convertStringToCamelCasesWithSpaces(
```
Do we really need both of these?

Can we do something like `cy.get('[data-qa-widget="..."]').within(() => { /* Assert the header text exists. */ })`?